src/RecoveryServices/RecoveryServices/help/Get-AzRecoveryServicesVaultProperty.md

```diff
@@ -25,13 +25,22 @@ The **Get-AzRecoveryServicesVaultProperty** cmdlet returns the properties of a R
 
 ### Example 1: Get Properties of a vault
 ```
-PS C:\> $vault = Get-AzRecoveryServicesVault -Name "MyVaultName"
+PS C:\> $vault = Get-AzRecoveryServicesVault -ResourceGroupName "resourceGroup" -Name "vaultName"
 PS C:\> $props = Get-AzRecoveryServicesVaultProperty -VaultId $vault.Id
 ```
 
 The first command gets a Vault object and then stores it in the $vault variable.
 The second command Gets the Vault Properties.
 
+### Example 2: Get CMK encryption Properties of a vault
+```
+PS C:\> $vault = Get-AzRecoveryServicesVault -ResourceGroupName "resourceGroup" -Name "vaultName"
+PS C:\> $encryption = Get-AzRecoveryServicesVaultProperty -VaultId $vault.Id -EncryptionSettings
```

Comment:
Encryption properties should be returned as part of the result of `Get-AzRecoveryServicesVaultProperty`. User should be able to get **all** the properties without needing to specify what to get.

Reply:
@isra-fel : Currently, encryption properties are returned via one API and other properties are called via another API. We needed to merge them together and we don't have time for this release. We do have a plan to integrate at an API level and the same API returning all properties. Until then, my proposal is to unblock customers by having this option and then later we don't need this (either by API merging or by PS client code merging)

Hope this makes sense.

Reply:
I understand your intention to make it work before merging the APIs. I'm just suggesting you to take a different way to do it:
`Get-AzRecoveryServicesVaultProperty` will
- call both APIs
- conbine the key properties into the vault properties
- write the vault properties as the output

This is for 2 reasons:
1. `-EncryptionSettings` will deprecate sooner or later. Removing it will be a breaking change. Why not we don't have it at the first place.
2. Changing the output of a cmdlet dramatically by the precense of a switch parameter makes it hard to use

Reply:
@isra-fel : I agree, but a small doubt - How can we modify the output type of this cmdlet (if we merge the output and create a new type)? won't it be a breaking change for customers using the existing parameter set?